packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/identity/overview-identity/overview-identity.html

```diff
@@ -0,0 +1,340 @@
+<div data-ng-if="!$ctrl.editMode">
```

Comment:
Why not extract all parts under `!$ctrl.editMode` in one component? (Also, button under the div L330 should be in this div element)